R/fmt.R

```diff
@@ -114,16 +110,15 @@ fmt_ci <- function(lower, upper, digits = 2, width = 3 + digits) {
 #'
 #' @return A numeric vector with the expected format.
 #'
-#' @importFrom dplyr if_else
-#'
 #' @export
 #'
 #' @examples
-#' fmt_pval(0.1234)
+#' fmt_pval(c(0.1234, 0.00002))
 fmt_pval <- function(p, digits = 3, width = 3 + digits) {
   scale <- 10^(-1 * digits)
   p_scale <- paste0("<", scale)
-  if_else(p < scale, p_scale,
+  ifelse(p < scale,
```

Comment:
We didn't need to use `dplyr::if_else()`, so this was a clean swap to remove the dependency. 
Added some more tests to verify correct behavior.

Reply:
Nice